### PR TITLE
Support standard kml files

### PR DESCRIPTION
**Pull request by sgonyea**
You may want make a larger bump to the version number if you merge this PR, as it potentially (not very likely) breaks KML files if they were generated very poorly and not in line with the KML standard:

https://developers.google.com/kml/documentation/kmlreference#coordinates

That said, this makes BorderPatrol more robust and less picky, so I really hope it gets merged.  Thank you for releasing this library!

### CODE REVIEW

**Comment by sgonyea** [comment]
Here is my use-case:

https://github.com/sgonyea/usa_kml_files

For anyone who needs this, while it is pending a merge, I've released the gem "border_patrol-sgonyea" with my changes.

File: spec/lib/border_patrol_spec.rb [hunk 1]
```diff
@@ -28,9 +39,8 @@
              <tessellate>1</tessellate>
              <coordinates>
                -10,25,0.000000
-               -1,30,0.000000
-               10,1,0.000000
-               0, -5,000000
+               -1,30,0.000000 10,1,0.000000
+               0,-5,000000
```
Comment (daicoden) [inline spec/lib/border_patrol_spec.rb:43]: Are all KML files guaranteed to not have spaces in the coordinates?  I thought this had been copied from a google example.

**Comment by sgonyea** [comment]
Hi @daicoden, if you go to this KML Validator:

http://kmlvalidator.com/validate.htm

and punch in this URL:

http://gist.github.com/sgonyea/5073176/raw/a72b23d45ed158b1aa19b85661817903bc644695/invalid.kml

Which contains the original coordinates from the spec... It'll come up the error:

```
kml:coordinates tuple is not 2D or 3D [4].
This relates to this OGC KML requirement:  Geometry-Coordinates
Please help us by providing your feedback on this error message.
```

The KML 2.2 Standard Doc states that there cannot be spaces in-between the tuples:

https://developers.google.com/kml/documentation/kmlreference#coordinates

So, I think it's a safe bet to not account for that -- but to also probably do a version bump, so anyone using a `~>` doesn't get burned.  You could also raise an exception in `BorderPatrol::Point` if x/y are nil, but that'd degrade performance.  It'll blow up anyway when calling `#contains_point?` and that should be a hint for them to validate their KML.

**Comment by daicoden** [comment]
Thanks @sgonyea!  Great idea on the version bump, go ahead and do it and let's do that and get this merged.  
Thank you!

**Comment by sgonyea** [comment]
@daicoden Done! Thanks.

**Pull request merged**